agenta-backend/agenta_backend/services/db_manager.py

```diff
@@ -671,6 +671,27 @@ async def create_app_and_envs(
         return app
 
 
+async def update_app(app_id: str, values_to_update: dict) -> None:
+    """Update the app in the database.
+
+    Arguments:
+        app_id (str): The app id
+        values_to_update (dict): The values to update in the app
+    """
+
+    async with db_engine.get_session() as session:
+        result = await session.execute(select(AppDB).filter_by(id=uuid.UUID(app_id)))
+        app = result.scalars().first()
+        if not app:
+            raise NoResultFound(f"App with {app_id} not found")
+
+        for key, value in values_to_update.items():
+            if getattr(app, key):
```

Comment:
Claude: 

> The getattr(app, key) check might not behave as intended. It will return False for attributes that exist but have falsy values (like 0, False, or None). This could prevent updating these values. Consider changing this to:
> ```
> if hasattr(app, key):
>     setattr(app, key, value)
> ```

Reply:
Ah, yes. That is true, thank you, Claude.

Reply:
Hold on. That's not `hasattr` in the code?!